src/index.test.ts

```diff
@@ -211,6 +211,37 @@ describe('stringify & parse', () => {
         },
       },
     },
+    'works for Errors with string cause': {
+      input: {
+        e: new Error('epic fail', { cause: 'no reason' }),
+      },
+      output: ({ e }: any) => {
+        expect(e.name).toBe('Error');
+        expect(e.message).toBe('epic fail');
+        expect(e.cause).toBe('no reason');
+      },
+      outputAnnotations: {
+        values: {
+          e: ['Error'],
+        },
+      },
+    },
+    'works for Errors with Error cause': {
+      input: {
+        e: new Error('epic fail', { cause: new Error('cause epic fail') }),
+      },
+      output: ({ e }: any) => {
+        expect(e.name).toBe('Error');
+        expect(e.message).toBe('epic fail');
+        expect(e.cause.name).toBe('Error');
```

Comment:
`e.cause` will still be of type `Error` here. But `output` should be a plain Javascript object, so that's a no-go. Look into how you can recurse through the error causes so you serialize everything.

To make it more obvious how the serialized output looks, please provide a plain object to `output` instead of a function.